simple-verifier/build.gradle

```diff
@@ -0,0 +1,86 @@
+
+    defaultConfig {
+        applicationId "com.example.simple_verifier"
+        minSdk 29
+        targetSdk 34
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary true
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.4.5'
+    }
+    packagingOptions {
+        resources {
+            excludes += '/META-INF/{AL2.0,LGPL2.1}'
+        }
+    }
+}
+
+dependencies {
+    implementation platform(libs.compose.bom)
+    implementation libs.bundles.androidx.core
+    implementation libs.bundles.androidx.lifecycle
+    implementation libs.bundles.androidx.navigation
+    implementation libs.bundles.androidx.crypto
+    implementation libs.bundles.bouncy.castle
+    implementation libs.bundles.compose
+
+
+    implementation("androidx.navigation:navigation-compose:2.6.0")
+
+    implementation("androidx.core:core-ktx:1.9.0")
+    implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.6.2")
+    implementation("androidx.activity:activity-compose:1.8.0")
+    implementation("androidx.activity:activity:1.6.0-alpha05")
+    implementation(platform("androidx.compose:compose-bom:2023.03.00"))
+    implementation("androidx.compose.ui:ui")
+    implementation("androidx.compose.ui:ui-graphics")
+    implementation("androidx.compose.ui:ui-tooling-preview")
+    implementation("androidx.compose.material:material-icons-extended")
+    implementation("androidx.compose.material3:material3")
+    implementation ("androidx.compose.material3:material3:1.2.0-alpha10")
+    implementation("com.android.identity:identity-credential-android:20231002")
+    implementation("com.android.identity:identity-credential:20231002")
+    implementation("com.google.android.material:material:1.10.0")
+    implementation libs.code.scanner
+//    implementation("org.bouncycastle:bcprov-jdk15on:1.70")
+//    implementation("org.bouncycastle:bcpkix-jdk15on:1.70")
+//    implementation("androidx.fragment:fragment-ktx:1.6.2")
+    testImplementation("junit:junit:4.13.2")
+    androidTestImplementation("androidx.test.ext:junit:1.1.5")
+    androidTestImplementation("androidx.test.espresso:espresso-core:3.5.1")
+    androidTestImplementation(platform("androidx.compose:compose-bom:2023.03.00"))
+    androidTestImplementation("androidx.compose.ui:ui-test-junit4")
+    debugImplementation("androidx.compose.ui:ui-tooling")
+    debugImplementation("androidx.compose.ui:ui-test-manifest")
+}
```

Comment:
Should use version catalogs instead of hardcoding artifacts and versions here. Just copy from the just-landed `wallet/build.gradle` instead.